docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  php:
+    build: .
+    environment:
+      SYMFONY_ENV: dev
+    volumes:
+      - ./:/srv/certificationy
+      - ./vendor:/srv/certificationy/vendor
```

Comment:
Yes of course, but if you have already `./:/srv/certificationy`, `./vendor:/srv/certificationy/vendor` is automatically included no?